CONTRIBUTING.md

```diff
@@ -37,3 +37,9 @@ In addition to the following guidelines, please follow the conventions already e
 
 - **Comments**:<br>
   Please use single-line comments to annotate significant additions, & [JSDoc-style](http://www.2ality.com/2011/08/jsdoc-intro.html) comments for new methods.
+
+Guidelines are enforced using JSSC.  This is a code style linter for programmatically enforcing style. To run the linter execute the following:
```

Comment:
Can you make JSSC a link to the npm package. No need for a description then as folks can click through.